README.md

```diff
@@ -7,17 +7,19 @@ Blueprint is a React-based UI toolkit for the web.
 It is optimized for building complex, data-dense web interfaces for _desktop applications_.
 If you rely heavily on mobile interactions and are looking for a mobile-first UI toolkit, this may not be for you.
 
+[**Read the introductory blog post ▸**](https://medium.com/@palantir/scaling-product-design-with-blueprint-25492827bb4a)
+
 [**View the full documentation ▸**](http://blueprintjs.com/docs)
 
-[**What's new in Blueprint 2.0 ▸**](https://github.com/palantir/blueprint/wiki/What's-new-in-Blueprint-2.0)
```

Comment:
not okay with losing this link. it should stay at the top of this page, i think.

Reply:
ok, seemed late enough. I'll move it to the section below. What's new + guide